build_projects/dotnet-host-build/NugetUtil.cs

```diff
@@ -8,25 +9,25 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public static class NuGetUtil
     {
+        public enum NuGetIncludePackageType
+        {
+            Standard = 1,
+            Symbols
```

Comment:
I think it's standard to specify each value when making flags (and marking `[Flags]`), since beyond `2` this doesn't work.